Makefile

```diff
@@ -25,10 +27,15 @@ docker-push:
 	  docker login; \
 	fi
 	docker tag smarkets/marge-bot:$$(cat version) smarkets/marge-bot:latest
```

Comment:
I think `:latest` should be `:stable` (meaning we can probably drop `:stable`, although I'm not opposed to it either). If you `docker run smarkets/marge-bot`, you should get a release version, IMO.